### PR TITLE
TINY-4713: Ensure tests aren't skipped due to parsing errors and add the symbol polyfill

### PR DESCRIPTION
**Pull request by lnewson**
We found out today that a number of tests weren't running on IE 11 (65 out of 782). This was due to `fast-check` being added as a dependency and it makes use of `Symbol` which is ES6 and as such was throwing an uncaught exception when importing the related tests on IE 11.

So this makes it so each test is wrapped in a try/catch and will now log a test failure instead. It also adds the `Symbol` polyfill from `core-js` to make it possible to run the tests on IE 11.

### CODE REVIEW

File: modules/server/src/main/ts/bedrock/compiler/Imports.ts [hunk 1]
```diff
@@ -15,17 +15,28 @@ const filePathToImport = (useRequire: boolean, scratchFile: string) => {
     filePath = filePath.replace(/\\/g, '/');
 
     return [
-      useRequire ? `require("${relativePath}");` : `import "${relativePath}";`, // rollup doesn't support require
-
-      `addTest("${filePath}");`
+      'try {',
```
Comment (ephox-mogran) [inline modules/server/src/main/ts/bedrock/compiler/Imports.ts]: How interesting. Is "```" an option?

File: modules/server/src/main/ts/bedrock/compiler/Imports.ts [hunk 2]
```diff
@@ -52,16 +63,20 @@ const addTest = (testFilePath: string) => {
     console.error('no test list to add tests to');
   }
 };
+const handleParseError = (testFilePath: string, error: any) => {
+  ${useRequire ? 'const UnitTest = require(\'@ephox/bedrock-client\').UnitTest;' : 'import { UnitTest } from \'@ephox/bedrock-client\';'}
```
Comment (ephox-mogran) [inline modules/server/src/main/ts/bedrock/compiler/Imports.ts]: Given the amount of escaping failures possible in statements like these, could we load these snippets from a file? Could that make it easier?

Comment (ephox-mogran) [reply modules/server/src/main/ts/bedrock/compiler/Imports.ts]: I guess the parameterisation of it probably makes it harder.

File: modules/server/src/main/ts/bedrock/compiler/Imports.ts [hunk 3]
```diff
@@ -85,10 +100,14 @@ var addTest = function (testFilePath) {
     console.error('no test list to add tests to');
   }
 };
+var handleParseError = function (testFilePath, error) {
```
Comment (ephox-mogran) [inline modules/server/src/main/ts/bedrock/compiler/Imports.ts]: This looks almost identical to the above. Can the string be shared?

Comment (lnewson) [reply modules/server/src/main/ts/bedrock/compiler/Imports.ts]: It's not identical at all, one is `JS` and the other is strict `TS`, so no they can't be shared sorry.

File: modules/server/src/main/ts/bedrock/compiler/Imports.ts (hunk 2 above)
Comment (lnewson) [reply modules/server/src/main/ts/bedrock/compiler/Imports.ts]: Yeah, most of this is dynamic and hence can't be in a file. We could look at making a template file for this, but I'm not convinced it's worth it as it'd still run into similar issues.

File: modules/server/src/main/ts/bedrock/compiler/Imports.ts (hunk 1 above)
Comment (lnewson) [reply modules/server/src/main/ts/bedrock/compiler/Imports.ts]: More than likely, this was probably from before ES 6 was possible on node and has just been adjusted slowly over time.

File: modules/server/src/main/ts/bedrock/compiler/Imports.ts (hunk 3 above)
Comment (ephox-mogran) [reply modules/server/src/main/ts/bedrock/compiler/Imports.ts]: Ah. I see.

Comment (ephox-mogran) [reply modules/server/src/main/ts/bedrock/compiler/Imports.ts]: What benefit do you get from making the TS one TS? Can it pick up any errors at that point? Would you be able to share it if they were both JS? Probably a stupid question, I'm just genuinely curious.

Comment (ephox-mogran) [reply modules/server/src/main/ts/bedrock/compiler/Imports.ts]: Or is the TS one required because there is a strict setting on?

Comment (lnewson) [reply modules/server/src/main/ts/bedrock/compiler/Imports.ts]: The TS one is required as bedrock reads from the projects tsconfig, so if strict is on it'll complain and fail to compile.

File: modules/server/src/main/ts/bedrock/compiler/Imports.ts [hunk 4]
```diff
@@ -52,16 +63,20 @@ const addTest = (testFilePath: string) => {
...
+const handleParseError = (testFilePath: string, error: any) => {
+  ${useRequire ? 'const UnitTest = require(\'@ephox/bedrock-client\').UnitTest;' : 'import { UnitTest } from \'@ephox/bedrock-client\';'}
+  UnitTest.test(testFilePath, () => { throw error; });
+};
 `,
     imports,
-    'export {};'
-  ].join('\n');
+    '\nexport {};'
+  ]).join('\n');
```
Comment (tinydylan) [inline modules/server/src/main/ts/bedrock/compiler/Imports.ts]: Would multiline interpolated strings help here?

**Pull request merged**